jetty-embedded-9/src/main/java/org/jboss/arquillian/container/jetty/embedded_9/ArquillianAppProvider.java

```diff
@@ -193,6 +196,8 @@ else if (context.toLowerCase(Locale.ENGLISH).startsWith("root-"))
         }
 
         webAppContext.setContextPath(context);
+        ServletHolder servletHolder = new ServletHolder(ServletMethodExecutor.ARQUILLIAN_SERVLET_NAME, ServletTestRunner.class);
```

Comment:
Ok so this wasn't picked up because I used incomplete web descriptor (having only "<web-app>" without further attributes) in Weld Servlet tests. I am going to update PR. Do you think it's possible to release new version then? I would like to run the tests with latest Jetty. Thank's a lot.